### ISSUE TITLE
CR Request for Accessibility Conformance Testing (ACT) Rules Format 1.1 - 

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/transitions#733]
# Document title, URLs, estimated publication date
[Accessibility Conformance Testing (ACT) Rules Format 1.1](https://www.w3.org/TR/2025/CR-act-rules-format-1.1-20250819/)

# Abstract
https://www.w3.org/TR/2025/CR-act-rules-format-1.1-20250819/#abstract

# Status
https://www.w3.org/TR/2025/CR-act-rules-format-1.1-20250819/#sotd

# Link to group's decision to request transition
https://www.w3.org/WAI/GL/wiki/Decisions

# Changes
[Changes since ACT Rules Format 1.1 First Public Working Draft](https://www.w3.org/TR/2025/CR-act-rules-format-1.1-20250819/#changes-since-FPWD)

# Requirements satisfied

# Dependencies met (or not)
[are your dependencies likely to change and impact your document?] No

# Wide Review

- [Tracking board](https://www.w3.org/PM/horizontal/review.html?shortname=act-rules-format)
- [Group issue tracker](https://github.com/w3c/wcag-act/issues/566)

# Issues addressed

[All ACT Rules Format 1.1 closed issues](https://github.com/w3c/wcag-act/issues?q=is%3Aissue%20state%3Aclosed)

# Formal Objections
No formal objections.

# Implementation

ACT rules contain test cases to illustrate passing, failing, and inapplicable accessibility examples. Accessibility tools and methodologies can compare their outcomes with those in ACT Rules test cases. Tools  expose their results via the [Evaluation and Report Language (EARL) 1.0 Schema](https://www.w3.org/TR/EARL10-Schema/).

The Act-Rules Community Group updated the existing rules so that they meet the requirements contained in ACT Rules Format 1.1 in [pr 2220](https://github.com/act-rules/act-rules.github.io/pull/2220).

For a list of implementations, see the [ACT implementations page](https://www.w3.org/WAI/standards-guidelines/act/implementations/).

# Patent disclosures
None.

**Comment by plehegar** [comment]
> Thanks [@plehegar](https://github.com/plehegar)
> 
> > Since we used to have "Previous Versions:" meaning previous versions of the same draft, your use to indicate the previous recommendation is a bit awkward. Please consider using "Previous Recommendation" instead.
> 
> I think it'd be easier at this point to change previous versions to https://www.w3.org/TR/2024/WD-act-rules-format-1.1-20240618/ Should be updated at https://www.w3.org/TR/2025/CR-act-rules-format-1.1-20250819/

That makes less sense to me since you already have the "History", which will direct you to all of the previous publications of the same draft. Either you change to Previous Recommendation, or simply remove the entry.

> > For the next transition:
> > 
> > 1. it should be clear if new tests were added between the 1.0 and 1.1 test suites and if any of the 1.0 tests were impacted outside of the change "Accessibility Support and Assumptions section are now subsections of Background.". Otherwise, it will be really difficult to evaluate simply using the [github PR diff](https://github.com/act-rules/act-rules.github.io/pull/2220/files#diff-9ed38e14ed8e076818a6cdd72f1658844d1fd8e893b98c7dfb6462ef5428363a).
> 
> When you say "Next transition", does it mean this very transition to CR or the next one to PR or REC?

Next one to start the AC Review. Not a blocker here, just wanted to set expectations for the future.

> We will put a dedicated page for the implementation report so that these things are clearer and easier to understand from the page prose rather than having to dig into the diff.

sounds great to me.
 
> > 3. You will need to [update references](https://labs.w3.org/normative-references/check?url=https%3A%2F%2Fwww.w3.org%2FTR%2F2025%2FCR-act-rules-format-1.1-20250819%2F). RFC7230 has been obsoleted. EPUB 3 points to idpf.org ?!? HTML reference should also be updated to point to https://html.spec.whatwg.org/multipage/
> 
> Done in [w3c/wcag-act#607](https://github.com/w3c/wcag-act/pull/607)

Thanks.

> > However, this [privacy comment](https://github.com/w3cping/privacy-request/issues/141#issuecomment-2498901571) didn't seem addressed and we were not able to find a rational for not adding it.
> 
> This issue was originally closed, so I had assumed that we were all happy with the current privacy section and there was hence no need to include this. In any case, this is now fixed in [w3c/wcag-act#606](https://github.com/w3c/wcag-act/pull/606)

Thank you.

Approved.

**Comment by plehegar** [comment]
One more **non-blocking** comment to consider:

> We noted "ACT Rules SHOULD use a [localizable](https://www.w3.org/TR/i18n-glossary/#dfn-localisation) format", which normatively links to the i18n glossary. While this is not a testable statement, we think this normative reference is still appropriate in this case.

After talking to more folks, one suggestion here would be
 s/ACT Rules SHOULD use a localizable format/ACT Rules should use a localizable format/

to avoid the potential misuse of RFC2119 (due to an untestable statement).

**Comment by daniel-montalvo** [comment]
> After talking to more folks, one suggestion here would be s/ACT Rules SHOULD use a localizable format/ACT Rules should use a localizable format/
> 
> to avoid the potential misuse of RFC2119 (due to an untestable statement).

Done in https://github.com/w3c/wcag-act/commit/482a43ed70e5807906c2770434312e659085adcc
